action/output.php

```diff
@@ -54,11 +38,11 @@ public function handle_read(Doku_Event $event, $param) {
      * @param $param
      */
     public function handle_output(Doku_Event $event, $param) {
-        global $ID;
-        if($this->lastread != $ID) return; // avoid nested calls
-        $this->lastread = '';
+        global $ID, $ACT;
+        if (act_clean($ACT) !== 'show') {
+            return;
```

Comment:
Mh, then we need to add this check in render, since otherwise we would get the output for `edit` `admin`, etc. as well.